Rakefile

```diff
@@ -82,6 +82,13 @@ task :cent_5rel_dock do
   sh "RS_SET=centos-7-docker INSTALL_METHOD=rpm RPM_REPO='opendaylight-5-release' bundle exec rake beaker"
 end
 
+# Deb VMs
+
+desc "Beaker tests against Ubuntu 16.04 VM with Boron release Deb"
+task :ubuntu_5rel_vm do
+  sh "RS_SET=ubuntu-16 INSTALL_METHOD=deb DEB_REPO='' bundle exec rake beaker"
```

Comment:
For the RPM, it's meant to be the param that determines which version of ODL to install. I called it "rpm repo" vs "rpm version" or "odl version" because we sometimes want something like the tip of a branch, vs a particular build. For example, 5-testing (latest Boron build, including snapshot and autorelease builds) vs 50-release (Boron 5.0.0 official release) vs 5-release (latest Boron official release/SR).

Not sure how well that translates into deb-world. Feel free to not use an exact mirror of this param, whatever makes sense.